src/System/Console/Hawk/Sandbox.hs

```diff
@@ -13,37 +13,36 @@
 --   limitations under the License.
 
 -- Extra steps to be performed if hawk was installed from a sandbox.
--- 
+--
 -- Extra steps are needed because the hawk binary needs runtime access
 -- to the hawk library, but the hint library only knows about the globally-
 -- installed libraries. If hawk has been installed with a sandbox, its
 -- binary and its library will be installed in a local folder instead of
 -- in the global location.
-{-# LANGUAGE TemplateHaskell, TupleSections #-}
+{-# LANGUAGE TemplateHaskell, TupleSections   #-}
```

Comment:
A mistake.  The editor first aligned the pragmas as

```haskell
{-# LANGUAGE TemplateHaskell #-}
{-# LANGUAGE TupleSections   #-}
```

I joined the two lines, forgot to remove the extra white-spaces after TupleSections.